### PR TITLE
Adding exception at used REST port problem

### PR DESCRIPTION
**Pull request by Patrikkk**

### CODE REVIEW

**Comment by WhiteXZ** [comment]
cc @nicatronTg @Enerdy @MarioE @tylerjwatson

File: TShockAPI/Rest/Rest.cs [hunk 1]
```diff
@@ -79,12 +79,20 @@ public Rest(IPAddress ip, int port)
 
 		public virtual void Start()
 		{
-			if (listener == null)
```
Comment (Enerdy) [inline TShockAPI/Rest/Rest.cs]: What happened to this? Is it always null or are there other reasons that deprecate this statement?

**Comment by nicatronTg** [comment]
Looks good to me!

**Comment by Enerdy** [comment]
:+1:

**Pull request merged**